exchanges/bybit/bybit_websocket.go

```diff
@@ -7,15 +7,16 @@ import (
 	"net/http"
```

Comment:
With this in your opener:
>This PR does not:
>    Attempt to fix subscription error handling

And no further details means I don't know what issue(s) you are referring to

Reply:
As such, here's the error I'm encountering
```
[DEBUG] | WEBSOCKET | 10/10/2024 09:08:14 | Bybit websocket connection: message received: {"req_id":"128873076","success":false,"ret_msg":"Batch subscription partially succeeded and partially failed.Successful subscriptions are as follows:[order position execution]. Subscription to the following topics failed because the topic does not exist or there is a subscription conflict:[myWallet.BTCUSDT].","op":"subscribe","conn_id":"cmjom4cfnontv8pqvlug-28whu5"}
```

Now you have to tell me if this is not an issue 😄

Reply:
This is an issue 😂 
It's due to the lack of a subscription name for `myWallet`.
What I don't understand at all is how you get a completely different error from me.
I get:
```
{"success":false,"ret_msg":"Invalid topic :[myWallet.BTCUSDT]","conn_id":"66a98329-eb29-480e-8423-5af14f82f730","req_id":"119798340","op":"subscribe"}
```

You got a partial subscription, but I got a failed subscription.
I caused mine by sending `myWallet.*` to a non-authed endpoint.
Did you get an ERROR message in your logs after your DEBUG message?

Fixed c95e40513

What I meant by my comment was that I'm not touching subscription mechanics at all until v5 lands, and maybe multi websockets. After that, I'll try to address partial subscribe failures.